Userland/Libraries/LibWeb/HTML/History.cpp

```diff
@@ -44,6 +44,52 @@ WebIDL::ExceptionOr<void> History::replace_state(JS::Value data, String const&,
     return shared_history_push_replace_state(data, url, IsPush::No);
 }
 
+WebIDL::ExceptionOr<u64> History::length() const
+{
+    // 1. Let document be this's associated Document.
```

Comment:
Incorrect comments here, these are not from https://html.spec.whatwg.org/multipage/history.html#dom-history-length